src/components/ComponentIndexPage/ComponentIndexList.js

```diff
@@ -1,62 +1,105 @@
-import { Link, Row, Column, Tag } from 'carbon-components-react';
+/* eslint-disable camelcase */
+/* eslint-disable global-require */
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable no-undef */
+import { Link, Row, Column, Tag, TooltipIcon } from 'carbon-components-react';
 import PropTypes from 'prop-types';
 import React from 'react';
 
 const componentImg = require('./images/placeholderCompIndex.svg');
+
 const href = '/get-started/about-carbon';
 
 function ComponentIndexList({ items }) {
+  let frameworkIcon;
+  try {
+    frameworkIcon = require(`./images/${framework}.svg`);
```

Comment:
I'd suggest moving these up where you have `componentImg` and do:

```js
const reactIcon = require('./images/React.svg');
const sketchIcon = require('./images/Sketch.svg');

const frameworkIcons = {
  React: reactIcon,
};
const designIcons = {
  Sketch: sketchIcon,
};
```

And then later on you can do:

```js
<img src={frameworkIcons[framework] || reactIcon} alt={framework} />
```

So that the requires don't kick-in on each render of the component